tobago-core/src/main/java/org/apache/myfaces/tobago/internal/renderkit/renderer/LinksRenderer.java

```diff
@@ -48,6 +49,9 @@ public void encodeBegin(final FacesContext facesContext, final UIComponent compo
         getExtraCssItem(),
         Orientation.vertical.equals(links.getOrientation()) ? BootstrapClass.FLEX_COLUMN : null,
         links.getCustomClass());
+    if (links.isDisabled()) {
```

Comment:
The HTML DIV tag includes the global attributes but no "disabled" attribute.
See: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/div

Imho the "disabled" attribute should not be rendered.